test/benchmark_http_client_test.cc

```diff
@@ -55,6 +55,16 @@ class BenchmarkClientHttpTest : public Test {
               {{":scheme", "http"}, {":method", "GET"}, {":path", "/"}, {":host", "localhost"}}));
       return std::make_unique<RequestImpl>(header);
     };
+    statistic_.connect_statistic = std::make_unique<StreamingStatistic>();
```

Comment:
Comment about having a constructor next to the struct itself applies here too. Assuming that the differences between the two call sites can be parameterized.

Reply:
Done.